policy/LsstSimMapper.yaml

```diff
@@ -291,3 +282,7 @@ datasets:
     python: builtins.str
     storage: TextStorage
     template: plots/%(filter)s/tract-%(tract)d/visit-%(visit)d/%(description)s.parq
+  jointcal_wcs:  # astrometric calibration produced by jointcal
+    template: 'jointcal-results/%(tract)04d/%(filter)s/jointcal_wcs-v%(visit)d_R%(raft)s_S%(sensor)s.fits'
+  jointcal_photoCalib:  # photometric calibration produced by jointcal
+    template: 'jointcal-results/%(tract)04d/%(filter)s/jointcal_photoCalib-v%(visit)d_R%(raft)s_S%(sensor)s.fits'
```

Comment:
Swap filter and tract in directory structure (as they have it for the coadd templates, e.g. 
```
deepCoadd_measMatch:
     template: deepCoadd-results/%(filter)s/%(tract)d/%(patch)s/srcMatch-%(filter)s-%(tract)d-%(patch)s.fits
```

I'd again move this up with the other **photoCalib** definition (line ~158).